### PR TITLE
Patch to Makefile and setup configuration for better testing environment

### PR DESCRIPTION
**Pull request by kmehant**
### Changes made
* Updated gitignore file for better development env
* Make fails as `pytest-3` command is not found so made patch to alternative choose `pytest` command instead
* Added `pytest-cov` plugin package to requirements as Makefile check uses it.

### CODE REVIEW

**Comment by kmehant** [comment]
@TomasTomecek Please review!

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,19 @@
+#test junk
 tests/data/basic_playbook.retry
 tests/data/basic_playbook_with_volume.retry
 contrib/ab-pod.yml
 .coverage.*
 .coverage*
+
+#python junk
```
Comment (TomasTomecek) [inline .gitignore]: can well call this with a different name than "junk", sounds a little offensive

how about "test artifacts" and "python artifacts"?

File: Makefile [hunk 2]
```diff
@@ -9,7 +9,7 @@ build-ab-img: recipe.yml
 	$(ANSIBLE_BENDER) build -- ./recipe.yml $(BASE_IMAGE) $(CONT_IMG)
 
 check:
-	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=yes pytest-3 --cov=ansible_bender -l -v $(TEST_TARGET)
+	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=yes pytest-3 --cov=ansible_bender -l -v $(TEST_TARGET) || PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=yes pytest --cov=ansible_bender -l -v $(TEST_TARGET)
```
Comment (TomasTomecek) [inline Makefile]: please make pytest a make variable and invoke this target using the variable:
```make
PYTEST_EXEC := pytest-3

...

check:
    PYTHONPATH=... $(PYTEST_EXEC) ...
```

File: setup.cfg [hunk 3]
```diff
@@ -47,6 +47,7 @@ install_requires =
 testing =
     pytest
     flexmock
+    pytest-cov
```
Comment (TomasTomecek) [inline setup.cfg]: +1

**Pull request merged**